src/test/Spec.hs

```diff
@@ -55,7 +55,7 @@ configTests = testGroup "Configuration tests" $
   , testCase "defaults.yaml" $ do
       EConfigWithUsage _ bad unset <- parseConfig "basic/default.yaml"
       assertBool ("unused options: " ++ show bad) $ null bad
-      let unset' = unset & sans "seed"
+      let unset' = unset & sans "seed" & sans "corpusDir"
```

Comment:
please update `default.yaml` and set `corpusDir: null`

Reply:
Fixed.